languages/cpp/generic/dune

```diff
@@ -8,6 +8,7 @@
    parser_cpp.ast
    tree-sitter-lang.cpp
    ast_generic
+   semgrep.configuring
```

Comment:
No, we don't want to depend on semgrep in languages/.
Maybe you can define a copy of the type in cpp_to_generic.mli
to avoid this dependency.
Note that if we use polymorphic variant like `AsVarFundef , you will not even need to write any conversion function.

Reply:
Done